Formula/fn.rb

```diff
@@ -0,0 +1,23 @@
+class Fn < Formula
+  desc "IronFunctions command-line tools"
+  homepage "https://iron.io"
+
+  stable do
+    url "https://github.com/iron-io/functions/releases/download/0.2.62/fn_mac"
+    sha256 "ba753fbaa4f9487c99f9fa5b870e8118ec2d8a558735ed3ce1144a19fda92460"
+    version "0.2.62-14"
```

Comment:
How is this `-14` part of the version known? It doesn't seem to be part of the release label. I think we'd want to stick with the publicly-visible version name; the `-14` seems to be an internal build detail.